add git_commit module (#673)
starship/starship: docs/config/README.md

truncation_symbol = ""
```

## Git Commit

> The `git_commit` module shows the active branch of the repo in your current directory.

Reviewer: I'd suggest adding a container clarifying that this module is disabled by default, as we do for the `time` module:
https://github.com/starship/starship/blob/ab310e2cf047e6ed57c7519d997d28e70e702fac/docs/config/README.md#L938-L943

truncation_symbol = ""
```

## Git Commit

The `git_commit` module shows the active branch of the repo in your current directory.

::: tip

This module is disabled by default.
To enable it, set `disabled` to `false` in your configuration file.

:::

### Options

| Variable              | Default   | Description                                      |
| ------------          | ----------| -------------------------------------------------|
| `style`               | `"green"` | The style for the module.                        |
| `prefix`              | `(`       | Prefix to display immediately before git commit. |
| `suffix`              | `)`       | Suffix to display immediately after git commit.  |
| `disabled`            | `true`    | Disables the `git_commit` module.                |

> | `commit_hash_length`  | `7`       | The length of the displayed git commit hash.     |

Reviewer: And here ☝️

> | `commit_hash_length` | `7`       | Minimal length for git commit hash to display    |

Reviewer: Nitpick: I find that this reads better.
```suggestion
| `commit_hash_length`     | `7`       | The length of the displayed git commit hash.        |
```
